go.mod

```diff
@@ -90,13 +90,13 @@ require (
 	gopkg.in/ini.v1 v1.64.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.22.3
-	k8s.io/apiextensions-apiserver v0.22.3
-	k8s.io/apimachinery v0.22.3
-	k8s.io/client-go v0.22.3
-	k8s.io/code-generator v0.22.3
+	k8s.io/api v0.24.0-alpha.0
+	k8s.io/apiextensions-apiserver v0.24.0-alpha.0
+	k8s.io/apimachinery v0.24.0-alpha.0
+	k8s.io/client-go v0.24.0-alpha.0
+	k8s.io/code-generator v0.24.0-alpha.0
```

Comment:
I was told the `v0.23.0-rc.0` tag is not yet available so this tag is used instead and corresponds to the same sha.